### PR TITLE
refactor: Disable Categories

### LINKED ISSUE #4101: 🔧 [Refactor] Disable Categories
## :zap: Refactor ticket

We have to disable Categories to be able to post again:
- Disable Categories as requirement in Posts
- Check if that will have an effect on the Dashboard/User Page etc

Most likely in a followup Ticket:
- What would be the use of Network specified Categories? Do we rly need them? @Tirokk 
- How to implement them? Maybe special tags could be up for the task?
- How would we filter/display those Categories?

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
We do not want to use categories anymore. So they are removed from the post contribution form and the post view. Please test!
For layout reasons I also removed the display of the language in the post view.

### Issues
- fixes #4101 

### Todo

- [ ] Please make a followup Issue to remove this Database field and remove data from the database in a migration. (See @ulfgebhardt comment below 👇🏼)
  - [ ] Issue for Categories out of Filter (See @ulfgebhardt comment below 👇🏼)
  - [x] Issue for Categories below Posts (See @ulfgebhardt comment below 👇🏼)
  - [ ] Issue for Categories out of Database (See @ulfgebhardt comment below 👇🏼)
  - [ ] Issue for Categories out of Admin page ( @Tirokk )
- [ ] What we do with the import of old posts? ( @Tirokk )
  

### Additional Informations

The "last“ version with categories is stored in branch `before-categories-switch--off`.

### CODE REVIEW

**Review by ulfgebhardt — CHANGES_REQUESTED** [review]
Create Post and View Post still don't work if I build this branch with docker.
I see you rebased it, but somehow the problem persists. Is it really fixed? @Mogge 
imagemehow

**Review by ulfgebhardt — APPROVED** [review]

**Comment by ulfgebhardt** [comment]
- [ ] Issue for Categories out of Filter
- [ ] Issue for Categories below Posts
- [ ] Issue for Categories out of Database

File: webapp/pages/post/_id/_slug/index.vue [hunk 1]
```diff
@@ -46,21 +46,6 @@
...
-          v-for="category in post.categories"
-          :key="category.id"
-          :icon="category.icon"
-          :name="$t(`contribution.category.name.${category.slug}`)"
-        />
-        <!-- Post language -->
-        <ds-tag v-if="post.language" class="category-tag language">
-          <base-icon name="globe" />
-          {{ post.language.toUpperCase() }}
-        </ds-tag>
```
Comment (Tirokk) [inline webapp/pages/post/_id/_slug/index.vue:62]: This is not related to this issue … 🤘🏼😇

**Review by Tirokk — APPROVED** [review]
Really cool @Mogge !!! 😎🚀🚀💫

Again a big step in direction of `wir.social` you coded! 🤗

PS: I added some notes into the main text for followup issues … ☝🏼

File: webapp/pages/post/_id/_slug/index.vue (hunk 1 above)
Comment (Mogge) [reply webapp/pages/post/_id/_slug/index.vue:62]: I know. But the layout is very ugly if we keep the language. As we will kick it as well, I think this is not an issue.
